hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/UserGroupInformation.java

```diff
@@ -529,6 +529,16 @@ private void setLogin(LoginContext login) {
     user.setLogin(login);
   }
 
+  /** This method checks for a successful Kerberos login
```

Comment:
This is generating a new JavaDoc warning:

```
[ERROR] /home/jenkins/jenkins-home/workspace/hadoop-multibranch_PR-5248/ubuntu-focal/src/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/UserGroupInformation.java:535: warning: no @return
[ERROR]   public boolean isLoginSuccess() {
[ERROR]                  ^
```

Additionally, I suggest sticking to the existing style of line break right after the opening and an asterisk on each line:

```
  /**
   * line 1
   * line 2
   *
   * @return foo
   */
```